### ISSUE TITLE
List things 'present in this work'

### ISSUE THREAD

**Issue by fuddl** [inventaire/inventaire#207]
how about listing _things_ that are present in a book. for instance [Prime Directive](https://www.wikidata.org/wiki/Q891149) is [_present in work_ (P1441)](https://www.wikidata.org/wiki/Property:P1441) in [_Over a Torrent Sea_](https://inventaire.io/entity/wd:Q54807276).

Here a quick sketch: 
![a manipulated screenshot of https://inventaire.io/entity/wd:Q54807276](https://user-images.githubusercontent.com/842548/43687179-3b4793d6-98d1-11e8-926b-4df7a317eee7.png)

A click on the _thing_ should reveal other works depicting the _thing_.

The list should be filtered by _things_ that are _characters_ which  are already listed under _characters_.

Any ideas about the label other than _things_?

**Comment by maxlath** [comment]
from [present in work (P1441)](https://www.wikidata.org/wiki/Property:P1441) claims, it seems it is supposed to be used as the inverse property of [characters (P674)](https://www.wikidata.org/wiki/Property:P674), so I would rather rely on a work P674 claims; the problem here being that I don't know what to use in place of P674 for non-human entities: would a Q54807276->P674->Q891149 statement make any sense?

**Comment by fuddl** [comment]
> would a Q54807276->P674->Q891149 statement make any sense?

No, because [Q891149](https://www.wikidata.org/wiki/Q891149) is not a character.
